src/gamma.jl

```diff
@@ -590,24 +590,45 @@ lgamma_r
     lfactorial(x)
 
 Compute the logarithmic factorial of a nonnegative integer `x`.
-Equivalent to [`lgamma`](@ref) of `x + 1`, but `lgamma` extends this function
+Equivalent to [`loggamma`](@ref) of `x + 1`, but `loggamma` extends this function
 to non-integer `x`.
 """
-lfactorial(x::Integer) = x < 0 ? throw(DomainError(x, "`x` must be non-negative.")) : lgamma(x + oneunit(x))
+lfactorial(x::Integer) = x < 0 ? throw(DomainError(x, "`x` must be non-negative.")) : loggamma(x + oneunit(x))
 Base.@deprecate lfact lfactorial
 
 """
-    lgamma(x)
+    logabsgamma(x)
```

Comment:
when a function returns a tuple, I like to write it as
```suggestion
    (y,s) = logabsgamma(x)
```